lib/ansible/module_utils/splitter.py

```diff
@@ -217,3 +219,13 @@ def unquote(data):
     if is_quoted(data):
         return data[1:-1]
     return data
+
+
+def stem(path):
```

Comment:
per discussion, you might want to look at is_subpath (ansible.utils.path) and also put this in common/file.py or create a common/path.py?